wrapper/src/main/java/software/amazon/jdbc/plugin/iam/LightRdsUtility.java

```diff
@@ -40,6 +40,17 @@ public class LightRdsUtility implements IamTokenUtility {
   // The time the IAM token is good for. https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.html
   private static final Duration EXPIRATION_DURATION = Duration.ofMinutes(15);
 
+  private final Clock clock;
+
+  public LightRdsUtility() {
+    this.clock = Clock.systemDefaultZone();
```

Comment:
why to switch from using `systemUTC()` to `systemDefaultZone`?

Reply:
Good point. Let me test it.

Reply:
Changed to UTC.